src/IconButton/index.scss

```
@@ -0,0 +1,58 @@
+.nds-icon-button {
+  background: none;
+  color: inherit;
+  border: none;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: var(--button-radius);
+  height: 32px;
+  width: 32px;
+
+  &.fontSize--l {
+    [class^="narmi-icon-"] {
+      transform: translate(0px, -1px);
+    }
+  }
+}
+
+.nds-icon-button--action {
+  --button-radius: 50%;
+  color: var(--color-secondary);
+
+  &:hover {
+    color: var(--theme-primary);
+    background-color: rgba(var(--theme-rgb-primary), var(--alpha-5));
+    text-decoration: none;
+  }
+}
+
+.nds-icon-button--plain {
+  height: 24px;
+  width: 24px;
+  color: var(--theme-secondary);
```

Same initial color for plain buttons:

```suggestion
  color: var(--theme-secondary);
```